sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormRecognizerClient.cs

```diff
@@ -74,13 +78,15 @@ public virtual RecognizeContentOperation StartRecognizeContent(Stream formFileSt
         [ForwardsClientCalls]
         public virtual async Task<RecognizeContentOperation> StartRecognizeContentAsync(Stream formFileStream, RecognizeOptions recognizeOptions = default, CancellationToken cancellationToken = default)
         {
-            await Task.Run(() => { }).ConfigureAwait(false);
-            throw new NotImplementedException();
-
             // TODO: automate content-type detection
             // https://github.com/Azure/azure-sdk-for-net/issues/10329
-            // ResponseWithHeaders<ServiceAnalyzeLayoutAsyncHeaders> response = ServiceClient.RestClient.AnalyzeLayoutAsync(contentType, formFileStream, cancellationToken);
-            // return new RecognizeContentOperation(ServiceClient, response.Headers.OperationLocation);
+            ResponseWithHeaders<ServiceAnalyzeLayoutAsyncHeaders> response = await ServiceClient.RestClient.AnalyzeLayoutAsyncAsync(ContentType.Pdf, formFileStream, cancellationToken).ConfigureAwait(false);
```

Comment:
Just a workaround while the new Service Client and content type detection are not in place.

Reply:
Do we have an issue tracking specifically to remove ContentType.Pdf?  (Or a comment on the bigger one?)  Will having this there break any tests?

Reply:
This won't break any tests since all tests we have for `RecognizeContent` operations use PDF files. I'll open an issue to make sure we don't forget it, but it should be closed soon.

Reply:
#11313